tests/rbd/image_with_metadata_feature_disable.py

```diff
@@ -0,0 +1,113 @@
+"""Module to verify disable image feature with meta-data set on image.
+
+Pre-requisites :
+We need cluster configured with atleast one client node with ceph-common package,
+conf and keyring files
+
+Test case covered -
+CEPH-9864 - Disable some existing Image Feature on Image having metadata set on it,
+with IO in progress from Client Side
+
+Test Case Flow:
+1. Create a Replicated pool
+2. Create an image
+3. Run Ios on an Image
+4. Set image meta on the image
+5. verify that image meta is set properly.
+6. Initiate IOs from client
+7. While running the IOs disable the existing image feature.
+8. Image features should be disable properly
+8. Repeat steps 1 to 7 for ecpool
+"""
+
+from datetime import datetime
+
+from ceph.parallel import parallel
+from tests.rbd.exceptions import RbdBaseException
+from tests.rbd.rbd_utils import initial_rbd_config
+from utility.log import Log
+from utility.utils import run_fio
+
+log = Log(__name__)
+
+
+def image_metadata_feature_disable(rbd, pool_type, **kw):
+    """Disable some existing Image Feature on Image having metadata set on it with IO in-progress.
+
+    Args:
+        rbd: RBD object
+        pool_type: pool type (ec_pool_config or rep_pool_config)
+        **kw: test data
+    """
+    pool = kw["config"][pool_type]["pool"]
+    image = kw["config"][pool_type]["image"]
+    feature_name = "deep-flatten"
```

Comment:
[Optional] Can you take feature name as input from test suite config? So that this module can be more generic, we can disable any required feature and check.

Reply:
Done.